packages/graphql-persisted/src/apollo.ts

```diff
@@ -32,7 +33,10 @@ export class PersistedLink extends ApolloLink {
     }
 
     return new Observable(observer => {
-      const {idFromOperation = defaultIdFromOperation} = this.options;
+      const {
+        alwaysIncludeQuery = false,
```

Comment:
I was proposing just `includeQuery`, since this directly maps to an option with that name in Apollo.

Reply:
That name seems confusing to me? It loses the explicitness that the query will always be included. I'm not sure what's really gained by excluding the word always. Especially since, as I pointed out, the Koa middleware uses always to describe similar behavior.